harmonise/zooma.py

```diff
@@ -0,0 +1,12 @@
+import requests
+
+
+class ZoomaClient:
```

Comment:
This class looks static, there are few ways we can improve this to make it better OOP
- the URL could be a class variable accepting in the constructor (eg. base_url)
- Rename get_json should have better name and could accept arguments (eg. field_label) and then construct the final API call url from base_url

Reply:
do you mean:

        field_label_json = zooma_client.field_label()

        if field_label_json is not None:
            for i, el in enumerate(field_label_json):
            
?